test/plugin/example/connector.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+
+	"github.com/cyberark/secretless-broker/pkg/secretless/log"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+)
+
+// SingleUseConnector creates an authenticated connection to a target TCP service.
+type SingleUseConnector struct {
+	logger log.Logger
+}
+
+// Connect is the function that implements the tcp.Connector func signature in this
+// example plugin. It has access to the client connection and the credentials (as a map),
+// and is expected to return the target service connection.
+//
+// This example connector works as follows:
+// 1. Waits for the initial message from the client
+// 2. Connect to a target service whose address is the value of the credential identified
+// by the key "address"
+// 3. Inject credentials from a credential identified by the key "auth"
+// 4. Write the initial message from the client with some modification
+func (connector *SingleUseConnector) Connect(
+	clientConn net.Conn,
+	credentialValuesByID connector.CredentialValuesByID,
+) (net.Conn, error) {
+	connector.logger.Debugln("waiting for initial write from client")
```

Comment:
Given the indentation of the method signature, it might be easier to read with a LF above this line

Reply:
`Debugln` should probably start capitalized (it's not an error message). Same applies to rest of non-error messages.

Reply:
will fix, but this was a copy-paste from the old example